### PR TITLE
PorousFlow contribution.

### PR DESCRIPTION
**Pull request by WilkAndy**
This removes unnecessary unit-test files and adds some utils.

Refs #6845

### CODE REVIEW

**Comment by WilkAndy** [comment]
Hopefully this will satisfy @dschwen :-)

**Comment by WilkAndy** [comment]
@cpgr - could you please check PorousFlowCapillaryPressureVG ?  I have modified it in many ways, and i hope it's what you intended:

 - std::max(pc, pc_max) is replaced by std::min(pc, pc_max) .  This means pc_max really is the *maximum* pc, which its name suggests

 - if (0 < sat_eff < 1) is replaced by explicit if (self >= 1), etc, and i explicitly return 0, pc_max, or pc when (i think) it's appropriate

 - added pc_max to dCap and d2Cap and returned the correct derivatives

 - fixed sign in the derivatives.  But, perhaps you wanted a negative sign there?

File: modules/porous_flow/include/utils/PorousFlowCapillaryPressureVG.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * van Genuchten capillary pressure for multiphase flow in porous media.
+ *
+ * Based on van Genuchten, M. Th., A closed for equation for
+ * predicting the hydraulic conductivity of unsaturated soils,
+ * Soil Sci. Soc., 44, 892-898 (1980).
+ */
+namespace PorousFlowCapillaryPressureVG
+{
+  /**
```
Comment (dschwen) [inline modules/porous_flow/include/utils/PorousFlowCapillaryPressureVG.h]: Namespaces are _not_ indented (http://mooseframework.org/wiki/CodeStandards/). Sorry, this must drive you guys crazy.

File: unit/src/PorousFlowEffectiveSaturationRSCTest.C [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "PorousFlowEffectiveSaturationRSCTest.h"
+
+CPPUNIT_TEST_SUITE_REGISTRATION( PorousFlowEffectiveSaturationRSCTest );
+
+PorousFlowEffectiveSaturationRSCTest::PorousFlowEffectiveSaturationRSCTest()
+{
+  _ep = 1.0E-8;
```
Comment (dschwen) [inline unit/src/PorousFlowEffectiveSaturationRSCTest.C]: could go in the initializer list

File: unit/src/PorousFlowEffectiveSaturationRSCTest.C [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+PorousFlowEffectiveSaturationRSCTest::~PorousFlowEffectiveSaturationRSCTest()
+{}
```
Comment (dschwen) [inline unit/src/PorousFlowEffectiveSaturationRSCTest.C]: brackets should be on separate lines. I'n most cases an empty virtual destructor is unnecessary. It is only needed in the top most base classes.

**Comment by dschwen** [comment]
That is a nice size. Looks really good! Just some level 100 nit-picking.

**Comment by permcody** [comment]
👍

**Pull request merged**